### DISCUSSION TITLE
为什么控制面板的密码socks认证信息会总是会变换呢？

### DISCUSSION THREAD

**Discussion by eightsheep67** [vernesong/OpenClash#3699] [General]
发现最近总是莫名秒控制面板被自己改成随机密码，然后socks认证信息自己添加新的clash加随机密码的账户，这个是新功能还是bug？？能关掉吗？

**Comment by eightsheep67** [comment]
而且这个配置应该是被接管了，即便我在配置中写了密码，还是会被系统修改成随机码。有什么地方能关掉吗？？我现在还没解决openwrt莫名自动重启的问题，重启完openclash就会生成随机码，很麻烦。

**Reply by ghost** [reply]
特性，通过面板目前没地方关

有能力就自己改 https://github.com/vernesong/OpenClash/commit/1891cc65f5b661e13987303fefc2416188713544 

https://github.com/vernesong/OpenClash/blob/1891cc65f5b661e13987303fefc2416188713544/luci-app-openclash/root/etc/uci-defaults/luci-openclash#L50-L55

否则就不要升级 157 之后的版本了

**Reply by eightsheep67** [reply]
是自己去对应目录删掉对应的行就行了吗

**Reply by ghost** [reply]
删了也行，我建议是把 password 固定下来，或者还原成之前版本的`只有新安装才会生成一个新密码`

**Reply by eightsheep67** [reply]
找到了，我先注释掉了。看看下次自动重启后还会不会变。
最近刚好在做pve下虚拟机的稳定测试，等它自动重启。

**Reply by ghost** [reply]
刚发现你还有`控制面板`这四个字，眼瞎

以防万一
https://github.com/vernesong/OpenClash/blob/1891cc65f5b661e13987303fefc2416188713544/luci-app-openclash/root/etc/uci-defaults/luci-openclash#L47-L55

---

<details>
<summary>再顺便，下面是156的写法</summary>

https://github.com/vernesong/OpenClash/blob/a646b47434ea9cceb35f4fdf6da49cb5f603110c/luci-app-openclash/root/etc/uci-defaults/luci-openclash#L47-L65

**Reply by eightsheep67** [reply]
大佬谢啦

**Reply by leeima** [reply]
> 大佬谢啦

求问，用的哪个写法？现在重启密码不变了吗？
